StarLight_Line/Class/IBaseWorking/SceneManager/Scene/ObjManager/Obj/MainSceneObj/Player/Player.cpp

```diff
@@ -0,0 +1,122 @@
+#include "Player.h"
+
+VOID Player::Init()
+{
+	m_PlayerPoint.x = m_MaxXArrayNum / 2;
+	m_PlayerPoint.y = m_MaxYArrayNum / 2;
+	m_Speed.x = m_Speed.y = 0.f;
+	m_PlayerPos.x = m_BasePos[m_PlayerPoint.y][m_PlayerPoint.x].x;	//�^�񒆂Ɏ��@��u��
```

Comment:
m_PlayerPointを変数を置いてください

Reply:
試した結果、こんな感じになりました。
```suggestion
VOID Player::Init()
{
	//長いので省略
	int* pPoX = &m_PlayerPoint.x;
	int* pPoY = &m_PlayerPoint.y;

	*pPoX = m_MAXXARRAYNUM / 2;
	*pPoY = m_MAXYARRAYNUM / 2;
	m_Speed.x = m_Speed.y = 0.0f;
	m_PlayerPos.x = m_BasePos[*pPoY][*pPoX].x;	//真ん中に自機を置く
	m_PlayerPos.y = m_BasePos[*pPoY][*pPoX].y;
}
```
Init()関数内は5行以内なのでわざわざ変数に置く必要性はないと思うのでやめておきますが、Update関数などのm_PlayerPointをたくさん使う場面では
buffer変数を使うようにしようと思います。